master/usage/troubleshooting/index.md

```diff
@@ -19,36 +38,64 @@ or you can set environment variables for `sudo` commands like this:
 ```
 
 Also be aware that connection information can be specified as a config
-file rather than using environment variables.  See the 
+file rather than using environment variables.  See the
 [Calicoctl Configuration Overview]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup)
 guide for details.
 
-## Ubuntu (or GNOME) NetworkManager
+## Container Connectivity Issues
 
-Disable [NetworkManager](https://help.ubuntu.com/community/NetworkManager) before
-attempting to use Calico networking.
+No matter what kind of connectivity issues you are having, we recommend you
+**run through all of the following checks in order to diagnose the problem.**
 
-NetworkManager manipulates the routing table for interfaces in the default network
-namespace where Calico veth pairs are anchored for connections to containers.
-This can interfere with the Calico agent's ability to route correctly.
+#### Host Can't Communicate with its Containers
 
-You can configure interfaces in the `/etc/network/interfaces` file if the
-NetworkManager removes your host's interfaces. See the Debian
-[NetworkConfiguration](https://wiki.debian.org/NetworkConfiguration)
-guide for more information.
+By default, hosts should be able to reach their running containers (even when policy would
+otherwise prevent it). Test this basic connection by pinging a container IP from its host.
+
+If this ping was not successful, check [Felix Logs](logging#felix) for issues.
+
+#### Containers on the Same Host Can't Communicate
+
```

Comment:
Wording of this section is odd.  And the bottom line seems to be that "if you can't communicate even though your policy is correct then your policy is not correct" :-)

Isn't this really just trying to say, if you expect to be able to communicate between containers on the same host and you can't then check your policy?